src/picom.c

```diff
@@ -726,21 +726,11 @@ static void configure_root(session_t *ps) {
 	}
 
 	log_info("Root configuration changed, new geometry: %dx%d", r->width, r->height);
-	bool has_root_change = false;
```

Comment:
We should keep this, there's no reason to remove support for backends without root change.

Reply:
What to do in case there is no root_change in the backend (xrender backend)?

Reply:
just let it keep doing what it's doing now.